polyfills/Array/prototype/@@iterator/detect.js

```diff
@@ -0,0 +1 @@
+typeof Array.prototype[Symbol.iterator] === 'function'
```

Comment:
This needs to be a bit more defensive - iterator may not exist on Symbol (we don't load the polyfill's dependencies for the detect)